cmd/server-main.go

```diff
@@ -250,6 +263,13 @@ func serverMain(ctx *cli.Context) {
 	// Initialize name space lock.
 	initNSLock(globalIsDistXL)
 
+	newObject, err := newObjectLayer(globalEndpoints)
```

Comment:
Moving this here won't work for distributed setup. For the disks to connect you need HTTP service to run first.

Reply:
moved back but changes were needed to newServerCacheObjects to accomodate the move back